hummingbot/connector/exchange/coinbase_advanced_trade/coinbase_advanced_trade_exchange.py

```diff
@@ -46,6 +47,12 @@
     from hummingbot.client.config.config_helpers import ClientConfigAdapter
 
 
+CoinbaseExchangeThrottler = AsyncThrottler(
+    rate_limits=constants.RATE_LIMITS,
+    limits_share_percentage=Decimal("100")
```

Comment:
this should use the config from the Hummingbot application, but as it's not widely used is okey to merge but I think that will be good to review it long term